packages/Button/src/Button.styles.js

```diff
@@ -4,9 +4,52 @@ import { css, keyframes } from "styled-components";
 import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
 import Kinds from "./ButtonKinds";
 
+// States
+
+const disabledStyles = css`
+  box-shadow: none;
+  color: ${tokens.color.blackDisabled};
+  cursor: not-allowed;
+  text-decoration: none;
+  text-shadow: none;
+
+  &:focus {
+    box-shadow: none;
+    text-shadow: none;
+    transform: none;
+  }
+
+  &,
```

Comment:
The lone `&` is to make these styles also apply to the "base" component?

Reply:
Correct.  Saves repeating the CSS rule.